pkg/tarmak/provider/amazon/amazon.go

```diff
@@ -356,6 +356,10 @@ func (a *Amazon) Validate() error {
 
 }
 
+func (a *Amazon) Verify() (result error) {
+	return result
```

Comment:
Am I right in saying this is the same as returning nil?

Reply:
I am https://play.golang.org/p/vJ5A24UnA99

I'm going to stick my neck out and say I prefer `return nil` :fire: 

```
func Verify() (error) {
	return nil
}
```

Reply:
(this all looks fine. I'm winding down for the day. Just ignore my wittering...)

Reply:
Yeah we use the same methodology on other places as well.